mamutils/sendreceivemam.go

```diff
@@ -0,0 +1,87 @@
+/*
+MIT License
+Copyright (c) 2017 Harry Boer
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+
+//Package mamutils is a utility to create Masked Authenticated Messages (MAM's)
+package mamutils
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/iotaledger/giota"
+)
+
+//ToMAMTrytes checks its validity and casts to giota.Trytes.
```

Comment:
The only thing this does is convert an ASCII (since everything above 255 is replaced by a space) string to trytes, so I'm not sure if this needs to be its own package since the same thing could be used for e.g. tags as well.

Reply:
No, indeed. I was wondering about this myself. So I will just use the file and take it out of the package.